bot/exts/info/doc/_cog.py

```diff
@@ -0,0 +1,442 @@
+        the first word of the name will be attempted to be used to get the item.
+        """
+        doc_item = self.doc_symbols.get(symbol_name)
+        if doc_item is None and " " in symbol_name:
+            symbol_name = symbol_name.split(" ", maxsplit=1)[0]
+            doc_item = self.doc_symbols.get(symbol_name)
+
+        return symbol_name, doc_item
+
+    async def get_symbol_markdown(self, doc_item: DocItem) -> str:
+        """
+        Get the Markdown from the symbol `doc_item` refers to.
+
+        First a redis lookup is attempted, if that fails the `item_fetcher`
+        is used to fetch the page and parse the HTML from it into Markdown.
+        """
+        markdown = await doc_cache.get(doc_item)
+
+        if markdown is None:
+            log.debug(f"Redis cache miss with {doc_item}.")
+            try:
+                markdown = await self.item_fetcher.get_markdown(doc_item)
+
+            except aiohttp.ClientError as e:
+                log.warning(f"A network error has occurred when requesting parsing of {doc_item}.", exc_info=e)
+                return "Unable to parse the requested symbol due to a network error."
+
+            except Exception:
+                log.exception(f"An unexpected error has occurred when requesting parsing of {doc_item}.")
+                return "Unable to parse the requested symbol due to an error."
+
+            if markdown is None:
+                return "Unable to parse the requested symbol."
+        return markdown
+
+    async def create_symbol_embed(self, symbol_name: str) -> Optional[discord.Embed]:
+        """
+        Attempt to scrape and fetch the data for the given `symbol_name`, and build an embed from its contents.
+
+        If the symbol is known, an Embed with documentation about it is returned.
+
+        First check the DocRedisCache before querying the cog's `BatchParser`.
+        """
+        log.trace(f"Building embed for symbol `{symbol_name}`")
+        if not self.refresh_event.is_set():
+            log.debug("Waiting for inventories to be refreshed before processing item.")
+            await self.refresh_event.wait()
+        # Ensure a refresh can't run in case of a context switch until the with block is exited
+        with self.symbol_get_event:
+            symbol_name, doc_item = self.get_symbol_item(symbol_name)
+            if doc_item is None:
+                log.debug("Symbol does not exist.")
+                return None
+
+            self.bot.stats.incr(f"doc_fetches.{doc_item.package}")
+
+            # Show all symbols with the same name that were renamed in the footer,
+            # with a max of 100 chars.
+            if symbol_name in self.renamed_symbols:
+                renamed_symbols = ", ".join(self.renamed_symbols[symbol_name])
+                footer_text = textwrap.shorten("Moved: " + renamed_symbols, 200, placeholder=" ...")
```

Comment:
```suggestion
                footer_text = textwrap.shorten("Similar: " + renamed_symbols, 200, placeholder=" ...")
```
The word `Moved:` does not convey the right information to me. Given that the reason it's there is when people get one similar term when they had another in mind, `similar` might make more sense.

Reply:
I didn't want to use similar here because the symbols the moved names refer to may be something completely unrelated to the symbol that's now referred to by the original name

Reply:
Isn't the move like how both Python and Django have a `help`, so you move Django's help to `django.help`? Assuming someone wanted Django's help and typed `!d g help`, doesn't it make more sense to have `similar`?

That is unless I got the idea wrong about the footer text meaning

Reply:
Yes, but the names may refer to something completely unrelated (apart from the name), even in the same inventory. For example [`list`](https://docs.python.org/3/library/stdtypes.html#list) which fetches the class and [`pdbcommand.list`](https://docs.python.org/3/library/pdb.html#pdbcommand-list) which gets you a debugger command that has nothing in common to the list class

Reply:
Ah, the reason I suggested `similar` was more of the similarity in name, not in function. After all, the reason it's there is because the names are overlapping, so `similar` isn't far fetched. I just feel `moved` might be too undescriptive.

Reply:
I'm not too invested in either of the terms so I'd like some other opinions; on one hand the names themselves are similar but then the docs they fetch usually are from some other module and can be something unrelated

Reply:
I didn't give it much thought before, but now I do think "moved" is unclear unless one understands how the internals work. Since "similar" is vague and may imply similarity in name or function, how about "Similar names:"?

Reply:
That sounds better, committed in b436dcf